ingest-app/src/cmr/ingest/services/ingest_service/subscription.clj

```diff
@@ -25,11 +29,48 @@
   [context concept]
   (let [metadata (:metadata concept)
         subscription (spec/parse-metadata context :subscription (:format concept) metadata)
-        concept (add-extra-fields-for-subscription context concept subscription)
+        concept (->> concept
+                     (add-extra-fields-for-subscription context subscription)
+                     (subscriptions/set-subscription-arn-if-applicable context :subscription))
         {:keys [concept-id revision-id]} (mdb2/save-concept
                                           context
                                           (assoc concept :provider-id (:provider-id concept)
                                                  :native-id (:native-id concept)))]
+    (subscriptions/add-or-delete-ingest-subscription-in-cache context concept)
     {:concept-id concept-id
      :native-id (:native-id concept)
      :revision-id revision-id}))
+
+(declare delete-subscription)
+#_{:clj-kondo/ignore [:unresolved-symbol]}
```

Comment:
I think there's a way to make this work for the `defn-timed` macro without needing to put this in every time it is used. I'll see if I can submit a simple PR for it.

Reply:
That would be great.  Its not because of the code or clojure, its because the linter doesn't understand the macro.